similar/main/ai.cpp

```diff
@@ -481,11 +481,25 @@ void ai_init_boss_for_ship(void)
 #endif
 }
 
+void boss_init_all_segments(objptridx_t boss_objnum)
```

Comment:
Ah yes, I think I used the `c` variant with another unrelated change. I probably tried it with these changes at some point but the compiler complained, because it was being written to.